src/scripts/scripts/zone/caverns_of_time/old_hillsbrad/boss_captain_skarloc.cpp

```diff
@@ -287,7 +287,7 @@ struct boss_captain_skarlocAI : public ScriptedAI
         else
             DevotionAura_Timer -= diff;
 
-        if (HeroicMode)
+        //Consecration (Heroic)
```

Comment:
not my favorite cuz:

`Consecration_Timer -= diff;`
a rare case (maybe) but it can happen that Consecration_Timer and diff are the same...